package.json

```diff
@@ -36,7 +36,11 @@
     "@babel/core": "^7.12.7",
     "@babel/preset-env": "^7.12.7",
     "@babel/preset-react": "^7.12.7",
+    "bootstrap": "^4.5.3",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-bootstrap": "^1.4.0",
+    "react-dom": "^17.0.1",
+    "react-router-dom": "^5.2.0",
+    "reactstrap": "^8.7.1"
```

Comment:
Are react-bootstrap and reactstrap packages adapting the Bootstrap framework to React? I've never heard of them before.